modules/openapi-generator/src/main/resources/swift5/modelObject.mustache

```diff
@@ -1,5 +1,5 @@
-{{^objcCompatible}}{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} {{#useClasses}}final class{{/useClasses}}{{^useClasses}}struct{{/useClasses}} {{{classname}}}: {{#useVapor}}Content{{/useVapor}}{{^useVapor}}Codable{{/useVapor}}{{#vendorExtensions.x-swift-hashable}}, Hashable{{/vendorExtensions.x-swift-hashable}} {
-{{/objcCompatible}}{{#objcCompatible}}@objc {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} class {{classname}}: NSObject, Codable {
+{{^objcCompatible}}{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} {{#useClasses}}final class{{/useClasses}}{{^useClasses}}struct{{/useClasses}} {{{classname}}}: {{#useVapor}}Content{{/useVapor}}{{^useVapor}}Codable, JSONEncodable{{/useVapor}}{{#vendorExtensions.x-swift-hashable}}, Hashable{{/vendorExtensions.x-swift-hashable}}{
```

Comment:
@0x0c another nit that I found is that at the end of the line, there is a space before the final `{` that was acidently removed and is creating some noise for example in the vapor client that shouldn't have any changes with this PR.
Could you please restore that space, to avoid noise in the generated code?
This command should generate the Swift sample projects.
```
./bin/generate-samples.sh bin/configs/swift5-*
`
``

Reply:
Done with https://github.com/OpenAPITools/openapi-generator/pull/11202/commits/672df35022bdcd38c5e291ae77ef9e198baf7cdb and https://github.com/OpenAPITools/openapi-generator/pull/11202/commits/8962b8b1c7d95bff5b4512bf5932f24cdd86bfb1